data/polySchema.json

```diff
@@ -0,0 +1,72 @@
+{ 
+	"$schema": "http://json-schema.org/draft-07/schema",
+	"type": "object",
+	"title": "Poly Genbank JSON schema",
+	"description": "The root schema of the Genbank JSON document",
+	"required": ["Meta","Features","Sequence"],
+	"additionalProperties": false,
+	"properties": {
+		"Meta": {"type": "object", "properties": {
+			"Name": {"type": "string"},
+			"GffVersion": {"type": "string"},
+			"RegionStart": {"type": "number"},
+			"RegionEnd": {"type": "number"},
+			"Size": {"type": "number"},
+			"Type": {"type": "string"},
+			"GenbankDivision": {"type": "string"},
+			"Date": {"type": "string"},
+			"Definition": {"type": "string"},
+			"Accession": {"type": "string"},
+			"Version": {"type": "string"},
+			"Keywords": {"type": "string"},
+			"Organism": {"type": "string"},
+			"Source": {"type": "string"},
+			"Origin": {"type": "string"},
+			"Locus": {"type": "object", "properties": {
+				"Name": {"type": "string"},
+				"SequenceLength": {"type": "string"},
+				"MoleculeType": {"type": "string", "enum": ["DNA","RNA"]},
```

Comment:
There's a ton of [official molecule types](https://github.com/TimothyStiles/poly/blob/0b37805da7c94831109b62298868be891859e411/io.go#L360) in io.go.

Reply:
Got it, and added.